blas/testblas/level1double.go

```diff
@@ -1902,7 +1902,7 @@ var DrotmgTests = []DrotmgTestStruct{
 		Name: "D1_Small_D2_Big_Flag_1",
```

Comment:
This seems to be named in opposite way.

Reply:
@btracey ?

Reply:
I'm trying to remember what I meant when I wrote that. I remember that I was working on this at my parents' house and trying to find cases to get 100% test coverage... My suspicion looking at it now is that Flag is a misnomer, and I suspect it is really referring to the `if absQ1 > absQ2` statement.

Reply:
I just read Dan and my emails (from 2013!) on Drotmg and while there was complaining about the documentation, there is no reference to what I meant by that `Flag` value. I still stand by my suspicion, and it seems to match when I look at a few of the values.